diff --git a/node_modules/azure-pipelines-tool-lib/tool.js b/node_modules/azure-pipelines-tool-lib/tool.js
index 81c72e9..79c2d27 100644
--- a/node_modules/azure-pipelines-tool-lib/tool.js
+++ b/node_modules/azure-pipelines-tool-lib/tool.js
@@ -19,7 +19,7 @@ const semver = require("semver");
 const util = require("util");
 const tl = require("azure-pipelines-task-lib/task");
 const cmp = require('semver-compare');
-const uuidV4 = require('uuid/v4');
+const uuidV4 = require('uuid').v4;
 const pkg = require('./package.json');
 const libJson = require('./lib.json');
 // English fallback messages for bundled scenarios where lib.json localization may not be available
